src/devices/src/virtio/block/mod.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod device;
+pub mod event_handler;
+pub mod request;
+
+pub use self::device::{build_config_space, Block};
+pub use self::event_handler::*;
+pub use self::request::*;
+
+use vm_memory::GuestMemoryError;
+
+pub const CONFIG_SPACE_SIZE: usize = 8;
+pub const SECTOR_SHIFT: u8 = 9;
+pub const SECTOR_SIZE: u64 = (0x01 as u64) << SECTOR_SHIFT;
+pub const QUEUE_SIZE: u16 = 256;
+pub const NUM_QUEUES: usize = 1;
+pub const QUEUE_SIZES: &[u16] = &[QUEUE_SIZE];
+
+#[derive(Debug)]
```

Comment:
Would be nice to `impl Display` for these (and other) errors originating in devices, that make their way to the end user. Out of scope for now, but I think we should have it.

Reply:
Let's do that after the merge.